resource_migrations/1.3.0.py

```diff
@@ -0,0 +1,111 @@
+import argparse
```

Comment:
It's very nice that you provide this migration option. There is a strong similarity to the case of an application using a DB. This makes me wonder: do you just want to call these migrations in either the entrypoint of the main container or a separate init container of the notebook-service?

Reply:
@ableuler I think an init container would make a lot of sense tbh. I will make that change.

Reply:
That way I also don't have to update the docs here and in renku on how to run such migrations. Also I can keep adding such migrations to this init container.

Reply:
Done @ableuler. I tested on my dev deployment a case where no patching is needed and a case where the project name needs to be patched. It worked.